### PR TITLE
Updated Docs about the installation of Meshery with Windows w/signoff

### LINKED ISSUE #3061: [Docs]- Publish information about solving the errors faced during deployment of Meshery for Windows
#### Current State
Meshery documentation does not mention the possible errors can that pop up while deployment of Meshery to the Windows platform. 

#### Desired State
          1.  Mentioning that it is not always necessary to install a distro for the deployment of Meshery.
          2.  Mentioning the errors that can be faced by a newbie while deploying Meshery for the first time.
          3.  Showing the possible ways for solving the errors.
          4.  Making the Meshery Doc for the Windows platform more beginner-friendly.

---

#### Contributor Resources

- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
- [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by debjani1413**
Signed-off-by: debjani1413 <deydebjani916@gmail.com>

**Description**
   Updated Docs about the installation of Meshery with Windows

This PR fixes #3061 
1. Mentions that it is not always necessary to install a distro for the deployment of Meshery.
2. Mentions the errors that can be faced by a newbie while deploying Meshery for the first time.
3. Showing the possible ways to solve the errors.
4. Made the Meshery Doc for the Windows platform more beginner-friendly.

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: docs/pages/installation/windows.md [hunk 1]
```diff
@@ -147,6 +150,15 @@ In this how-to, [K3d](https://github.com/rancher/k3d) will be used as it relies
   </div></div>
   </pre>
 
+If using scoop, run the following in the powershell to install  a Kubernetes cluster :
```
Comment (leecalcote) [inline docs/pages/installation/windows.md:153]: Capitalize Scoop and PowerShell

File: docs/pages/installation/windows.md [hunk 2]
```diff
@@ -156,3 +168,29 @@ Follow the [installation steps]({{ site.baseurl }}/installation#windows) to inst
...
+
+2. Change your current context to docker-desktop:
+
+  <pre class="codeblock-pre">
+  <div class="codeblock"><div class="clipboardjs">
+  kubectl config use-context docker-desktop
+  </div></div>
+  </pre>
+
+3. Restart mesheryctl.
```
Comment (leecalcote) [inline docs/pages/installation/windows.md:194]: Restart Meshery (not mesheryctl)
```
<pre class="codeblock-pre">
  <div class="codeblock"><div class="clipboardjs">
  mesheryctl system restart
  </div></div>
</pre>
```

File: docs/pages/installation/windows.md [hunk 3]
```diff
@@ -71,6 +71,7 @@ Open Powershell in administrator mode and run:
 Set the default version to *WSL2*, which will be inherited by any distro you wish to use.
 
 Enable VM (Virtual Machine) feature:
+Open Powershell in administrator mode and run:
```
Comment (leecalcote) [inline docs/pages/installation/windows.md:74]: PowerShell

File: docs/pages/installation/windows.md [hunk 4]
```diff
@@ -137,6 +139,7 @@ The Docker Desktop application for Windows includes a comprehensive set of tools
 ### 4. <a name="step4"> <b>Install a Kubernetes cluster</b></a>
 
 Once Docker is installed, the next step will be to install a Kubernetes cluster.
+Under settings in the Docker Desktop application, enable *Kubernetes*.
```
Comment (leecalcote) [inline docs/pages/installation/windows.md:142]: "Under "Settings" in Docker Desktop, ..."

**Review by leecalcote — COMMENTED** [review]
Thank you for working through this, @debjani1413!

**Comment by debjani1413** [comment]
@leecalcote done the required changes do let me know

**Review by chandrashritii — APPROVED** [review]

**Comment by chandrashritii** [comment]
Looks good :) @debjani1413

**Pull request merged**